pulsar/build.rs

```diff
@@ -0,0 +1,5 @@
+fn main() {
+    let output = std::process::Command::new("git").args(["rev-parse", "HEAD"]).output().unwrap();
+    let git_hash = String::from_utf8(output.stdout).unwrap();
+    println!("cargo:rustc-env=GIT_HASH={git_hash}");
+}
```

Comment:
Why is this necessary?

Reply:
This is actually not new file, but a move of original `build.rs`. I forgot to remove the original file in the main directory.  I will remove the root file and amend the commit, so it will be registered as move.

It is used to set the implementation version of substrate node.